Data/Sys/Shaders/default_pre_post_process.glsl

```diff
@@ -293,6 +299,33 @@ float4 AreaSampling(float3 uvw, float gamma)
 	return avg_color / (area_corners + area_edges + area_center);
 }
 
+/***** 4x Rotated Grid SuperSampling *****/
+
+// Based on 4-Rook Antialiasing (RGSS) by Alan Wolfe
+// https://www.shadertoy.com/view/llj3Dw
+// https://blog.demofox.org/2015/04/23/4-rook-antialiasing-rgss/
+
+float4 RGSS(float3 uvw, float gamma)
+{
+	float2 step = GetInvWindowResolution();
+	float4 color = float4(0);
+
+	// | | |*| |
+	// |*| | | |
+	// | | | |*|
+	// | |*| | |
+
+	float s = 1.0/8.0;
+	float l = 3.0/8.0;
+
+	color += QuickSampleLinear(uvw.xy + float2( s, l) * step, uvw.z, gamma);
+	color += QuickSampleLinear(uvw.xy + float2( l,-s) * step, uvw.z, gamma);
```

Comment:
For this to be completely accurate you should be using our own bilinear implementation, as it's gamma corrected before averaging the 4 samples, not after (gamma space colors can't be blended correctly).
'BilinearSample()'

Reply:
Right, forgot about that. I'd rather keep it sharper and cheap since there's the Area filter for smoothness.

Reply:
I don't think it would affect the sharpness, if anything, gamma correctness would make it sharper I think.
In any case, it's more about keeping the code more in line with the other methods, as of now they are all gamma corrected, so it would be cool if this one was as well. The performance cost is probably trivial.
That said, if you really don't want to change it, just add a comment on top explaining why (and rename `QuickSampleLinear` to `QuickSampleBilinear` because linear seems to imply it returns a linear color :) ).